crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1048,22 +1032,34 @@ impl<P: PhaseItem, const I: usize> RenderCommand<P> for SetMeshBindGroup<I> {
         let Some(mesh) = mesh_instances.get(entity) else {
             return RenderCommandResult::Success;
         };
-        let skin_index = skin_indices.get(entity);
-        let morph_index = morph_indices.get(entity);
+        let skin_index = skin_indices.current.get(entity);
+        let morph_index = morph_indices.current.get(entity);
+        let mut previous_skin_index = None;
+        let mut previous_morph_index = None;
 
         let is_skinned = skin_index.is_some();
-        let is_morphed = morph_index.is_some();
-
-        let Some(bind_group) = bind_groups.get(mesh.mesh_asset_id, is_skinned, is_morphed) else {
+        let morph_id = morph_index.is_some().then_some(mesh.mesh_asset_id);
+        let mut is_motion_vectors = PREPASS && has_motion_vectors;
+
+        if is_motion_vectors {
+            // disable is_motion_vectors if this mesh doesn't have a previous value for
+            // skins or morphs. This way, we get the shader variant without the bindings for
+            // the previous value, which doesn't exist.
```

Comment:
Could this be simplified with a dummy binding? That is, we bind a fallback buffer (like the fallback images), but we don't use it?

Reply:
It would be great. Though, at the cost of pushing the "is motion vector" info to a uniform, add the is_motion_vector flag to the uniform for each mesh, enable that uniform conditionally on `MOTION_VECTOR_PREPASS` And not read the motion vector specific buffers when the bit is off.

This would allow getting rid of the combinatorial shader variants too, as we could set unconditionally the previous_joint_matrices/previous_morph_weight buffers.

Though no idea how much work & code that is.